Azure/communication-ui-library-android
# [Feature] additional baseline test cases

## Purpose

* add 2 network loss test cases to the baseline set
* refactoring to make code cleaner

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

```
Verify that the following are valid
* generate ACS Token
* paste token into `TestFixtures.acsToken`
* connect emulator or physical device
* ./gradlew  cDAT
```

## What to Check
* all test cases pass
